codebleu/parser/DFG.py

```diff
@@ -1213,3 +1213,176 @@ def DFG_javascript(root_node, index_to_code, states):
                 DFG += temp
 
         return sorted(DFG, key=lambda x: x[1]), states
+
+
+def DFG_rust(root_node, index_to_code, states):
```

Comment:
Please check pipeline failed, you need to apply `black` formatter (`python -m black .`)
Also pipeline if checking isort/ruff/mypy